src/components/CheckIn/CheckInItemsWrapper.js

```diff
@@ -126,6 +135,7 @@ class CheckInItemsWrapper extends Component {
           locations={locations}
           poLineOrderFormat={poLineOrderFormat}
           stripes={stripes}
+          createInventoryValues={this.getCreateInventoryValues()}
```

Comment:
the whole line will be passed down, since there are a lot of PO Line attributes are used below. please check https://github.com/folio-org/ui-orders/pull/367/files#diff-80d88b4db7f4f91f139f1fbf10874406R46

Reply:
Got it. Updated